### PR TITLE
Add blog post for External Resources

### PR DESCRIPTION
**Pull request by KarmaGYZ**
Apache Flink 1.11 introduces a new [External Resource Framework](https://ci.apache.org/projects/flink/flink-docs-master/ops/external_resources.html),
which allows you to request external resources from the underlying resource management systems (e.g., Kubernetes) and accelerate your workload with
those resources. As Flink provides a first-party GPU plugin at the moment, we will take GPU as an example and show how it affects Flink applications
in the AI field. Other external resources (e.g. RDMA and SSD) can also be supported [in a pluggable manner](https://ci.apache.org/projects/flink/flink-docs-master/ops/external_resources.html#implement-a-plugin-for-your-custom-resource-type).

### CODE REVIEW

**Comment by KarmaGYZ** [comment]
cc @tillrohrmann @MarkSfik

**Review by tillrohrmann — APPROVED** [review]
Thanks for creating this PR @KarmaGYZ. LGTM. Waiting for @MarkSfik before merging it.

File: _posts/2020-08-03-external-resource.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+Apache Flink 1.11 introduces a new [External Resource Framework](https://ci.apache.org/projects/flink/flink-docs-master/ops/external_resources.html),
+which allows you to request external resources from the underlying resource management systems (e.g., Kubernetes) and accelerate your workload with
+those resources. As Flink provides a first-party GPU plugin at the moment, we will take GPU as an example and show how it affects Flink applications
+in the AI field. Other external resources (e.g. RDMA and SSD) can also be supported [in a pluggable manner](https://ci.apache.org/projects/flink/flink-docs-master/ops/external_resources.html#implement-a-plugin-for-your-custom-resource-type).
+
+# End-to-end real-time AI with GPU
+
+Recently, AI and Machine Learning have gained additional popularity and have been widely used in various scenarios, such
+as personalized recommendation and image recognition. [Flink](https://flink.apache.org/), with the ability to support GPU
```
Comment (MarkSfik) [inline _posts/2020-08-03-external-resource.md:20]:
```suggestion
as personalized recommendation and image recognition. [Flink](https://flink.apache.org/), with its ability to support GPU
```

**Comment by MarkSfik** [comment]
Thanks for the pull request @KarmaGYZ and for checking @tillrohrmann 
The post looks good to me. 
Markos

**Pull request closed without merging**